src/main/java/com/synopsys/integration/jira/common/cloud/model/IssueSearchResponseModel.java

```diff
@@ -35,6 +35,17 @@
     private Object names; // TODO create a bean for this object
```

Comment:
While we're at it, can we replace an `Object` with `JsonElement`? That way, if we ever need to access these fields, we don't necessarily need to wait for a new version of the library to be released.